dpp-backend/scripts/requirements.txt

```diff
@@ -0,0 +1 @@
+requests==2.31.0
```

Comment:
We need Ip check here

Reply:
[DEPENDENCIES_BACKEND_SCRIPTS.txt](https://github.com/user-attachments/files/15902785/DEPENDENCIES_BACKEND_SCRIPTS.txt)

Reply:
you can use it and remove the txt